CountryPicker/CountryPicker/CountryManager.swift

```diff
@@ -31,35 +33,27 @@ open class CountryManager {
         }
         return nil
     }
-
+    
     private init() {
     }
-
+    
     func loadCountries() throws {
         let bundle = Bundle(for: CountryManager.self)
-        if let countriesPath = bundle.path(forResource: "CountryPickerController.bundle/countries", ofType: "plist") {
-            if let  array = (NSArray(contentsOfFile: countriesPath) as? [String]) {
-                self.countries.removeAll()
-                for item in array {
-                    let country = Country(countryCode: item)
-                    self.countries.append(country)
-                }
-                self.countries = self.countries.sorted { $0.countryName < $1.countryName }
-            } else {
-                throw "Missing array of countries plist in CountryPicker"
+        if let countriesPath = bundle.path(forResource: "CountryPickerController.bundle/countries", ofType: "plist"),
+            let  array = (NSArray(contentsOfFile: countriesPath) as? [String]) {
+            self.countries.removeAll()
+            array.forEach {
+                self.countries.append(Country(countryCode: $0))
             }
+            self.countries = self.countries.sorted { $0.countryName < $1.countryName }
         } else {
-            throw "Countries could not be loaded from plist , please check path"
+            throw "Missing array of countries plist in CountryPicker"
         }
     }
 
     func allCountries() -> [Country] {
         return countries
     }
-
-    class func country(with code: String) -> Country {
```

Comment:
Please tell why you consider this function to be deleted.

Reply:
I couldn't find any uses of it across the project?

Reply:
Cool. Great 💯

Reply:
🔥👌🏼